### PR TITLE
Monitor the Docker containers through the cAdvisor REST API.

### PR DESCRIPTION
**Pull request by RazeemM**

### CODE REVIEW

File: templates/cAdvisor/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Docker Monitoring using cAdvisor
+================================
+
+This template provides monitoring of the docker containers and file systems using cAdvisor.
```
Comment (bertysentry) [inline templates/cAdvisor/README.md:4]: Please add a link to cAdvisor:

```md
...file systems using [cAdvisor](https://github.com/google/cadvisor).
```

**Review by bertysentry — APPROVED** [review]

File: templates/cAdvisor/README.md (hunk 1 above)
Comment (RazeemM) [reply templates/cAdvisor/README.md]: Done!

**Review by bertysentry — APPROVED** [review]

File: templates/cAdvisor/cadvisor.cfg [hunk 2]
```diff
@@ -0,0 +1,1360 @@
...
+"/SENTRY/X/cadvisor:containers/parameterList" = { REPLACE = "ExecutionTime;0,HTTPStatusCode;0,Result;1,Status;1" },
+"/SENTRY/X/cadvisor:containers/parent" = { REPLACE = "cadvisor" },
+"/SENTRY/X/cadvisor:containers/pollingIntervalList" = { REPLACE = "httpRequestColl;continuous;120;" },
+"/SENTRY/X/cadvisor:containers/productVersion" = { REPLACE = "10300" },
+"/SENTRY/X/cadvisor:containers/reportAsCollectionError" = { REPLACE = "0" },
+"/SENTRY/X/cadvisor:containers/successfulHttpStatusCode" = { REPLACE = "200-208,226" },
+"/SENTRY/X/cadvisor:containers/thresholdList" = { REPLACE = "ExecutionTime;0 0 0 0 0 0;1 15 30 0 0 1;1 30 2147483646 0 0 2,Status;0 0 0 0 0 0;1 1 1 0 0 1;1 2 2 0 0 2" },
+"/SENTRY/X/cadvisor:containers/timeout" = { REPLACE = "30" },
+"/SENTRY/X/cadvisor:containers/type" = { REPLACE = "HTTPRequest" },
+"/SENTRY/X/cadvisor:containers/url" = { REPLACE = "http://%{HOSTNAME}:%{PORT}/api/v1.3/containers" },
```
Comment (Raphyyyy) [inline templates/cAdvisor/cadvisor.cfg:37]: This template is used for monitoring docker container right?

We should use /api/v1.3/containers/docker if we want to compute docker container only:
_"Note that the root container (/) contains usage for the entire machine. All Docker containers are listed under /docker. Also, type=name is not required in the examples above as name is the default type."_ (https://github.com/google/cadvisor/blob/master/docs/api_v2.md)

Comment (RazeemM) [reply templates/cAdvisor/cadvisor.cfg:37]: API v1.3 is better than v2 to get stats for each docker. I tried v2 and then went back to 1.3. Regarding Type, in a real user environment there may be different types. We cannot design the template suitable only to our env.

**Comment by Raphyyyy** [comment]
In overall, why don't you convert bytes to megabytes or gigabytes ?? For example : memory metrics are computed in bytes

**Comment by Raphyyyy** [comment]
Talked with @RazeemM  and everything is ok

**Review by MarcDouguedroit — APPROVED** [review]
Very nice!

**Pull request merged**